e2e/README.md

```diff
@@ -35,6 +39,38 @@ In the e2e folder you can invoke tests either with `make test` or with:
 To run both the build and test step you can invoke `make e2e` from the parent
 directory.
 
+To run the end 2 end test and keep the kind state execute:
+```bash
+NOCLEANUP=True ./run.sh
+```
+
+## Run indidual test
+
+After having executed a normal E2E run with `NOCLEANUP=True` Kind still continues to run, allowing you subsequent test runs.
+
+To run an individual test, run the following command in the `e2e` directory
```

Comment:
This may or may not work depending on the state in which the last test to run leaves the cluster.
It is assumed that individual unit tests will properly clean up after themselves, but that is not given.

Reply:
Yes, you are correct. This does not work when stuff is broken. But we cant fix it all now. Future work to have e2e tests not be so mixed up and depending on each other.

Most are easy, bit more annoying to fix are those nodes label and toleration changes.